src/app/core/theme.component.ts

```diff
@@ -2,15 +2,24 @@ import { Component } from '@angular/core';
 import { PowerIconComponent } from '@shared/ui';
 import { SpinnerDotsComponent } from '@shared/index';
 import { CircularButtonComponent } from '@shared/ui/circular-button/circular-button.component';
+import { NavbarComponent } from '../features/ui/navbar/navbar.component';
```

Comment:
Navbar importowany bezpośrednio. Powinien zamiast tego być dodany to index.ts i importowany jako featured/ui

Reply:
Done